Terminal.Gui/Application/Application.Initialization.cs

```diff
@@ -163,10 +163,7 @@ internal static void InternalInit (
                                                 );
         }
 
-        Driver.SizeChanged += Driver_SizeChanged;
-        Driver.KeyDown += Driver_KeyDown;
-        Driver.KeyUp += Driver_KeyUp;
-        Driver.MouseEvent += Driver_MouseEvent;
+        InitState ();
```

Comment:
1) Please rename this to `SubscribeDriverEvents`
2) Move the call up to line 153 (just after the call to `Driver!.init()`)
3) Add another `internal` method `UnsubscribeDriverEvents` that does the `-=` and replace the code in `Application.cs` line 180 with a call to this new method.